Classes/Event/NewsInitializeActionEvent.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+namespace GeorgRinger\News\Event;
+
+use GeorgRinger\News\Controller\NewsController;
+
+/**
+ * This file is part of the "news" Extension for TYPO3 CMS.
+ *
+ * For the full copyright and license information, please read the
+ * LICENSE.txt file that was distributed with this source code.
+ */
+final class NewsInitializeActionEvent
+{
+    /**
+     * @var NewsController
+     */
+    private $newsController;
```

Comment:
use type and remove comment, also below

Reply:
i copied the structure (for this and all the other suggestion that you made) from the already created events. Should i make a new branch for it? Because every event does not have `declare(strict_types=1);` and the comments are there for every getter und setter